### PR TITLE
Azure Stack Build Update

### PR DESCRIPTION
**Pull request by deathly809**
## Description

@markcowl / @cormacpayne 

I updated the build process to work with the upcoming Azure Stack Admin modules.  I have a ps-sign job which completed successfully here,

https://azuresdkci.westus2.cloudapp.azure.com/view/PowerShell/job/ps-sign/401/

## Checklist

- [X] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md)
- [X] The title of the PR is clear and informative
- [ ] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#updating-the-change-log)
- [X] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/preview/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr)
    - [ ] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/help-generation.md)

### CODE REVIEW

**Comment by bganapa** [comment]
@deathly809  the file changes are huge as you have release branch changes? In that case we need to wait till release getting merged in to preview..

File: build.proj [hunk 1]
```diff
@@ -25,53 +29,83 @@
     /p:Scope
       'ServiceManagement': service management
       'AzureStorage': storage data plane cmdlets
+      'Stack': Azure Stack cmdlets
       'Subfolder under src\ResourceManager': An individual cmdlet module
-      By default, it builds all
+      By default, it builds everything but Stack
```
Comment (bganapa) [inline build.proj]: psl update the comment, we build all by default now

File: tools/PublishModules.ps1 [hunk 2]
```diff
@@ -12,296 +12,757 @@
...
+    PROCESS {
+        $regex = New-Object System.Text.RegularExpressions.Regex "RequiredModules\s*=\s*@\([^\)]+\)"
+        $content = (Get-Content -Path $Path) -join "`r`n"
+        $text = $regex.Replace($content, "RequiredModules = @()")
+        Out-FileNoBom -File $Path -Text $text
+
+        $regex = New-Object System.Text.RegularExpressions.Regex "NestedModules\s*=\s*@\([^\)]+\)"
+        $content = (Get-Content -Path $Path) -join "`r`n"
+        $text = $regex.Replace($content, "NestedModules = @()")
+        $text | Out-File -FilePath $Path
```
[the remark this answers is not in the document]
Comment (deathly809) [reply tools/PublishModules.ps1]: Fixed.  Thanks.

**Comment by deathly809** [comment]
@markcowl  The only changes I could find was that I removed myself from the createmappings_rules.json.  There might have been other changes but I just can't seem to find them.

**Comment by deathly809** [comment]
https://github.com/Azure/azure-powershell/pull/5984/

**Pull request closed without merging**